source/libraries/ContractDeployer.ts

```diff
@@ -317,55 +251,27 @@ export class ContractDeployer {
-        if (!marketAddress) {
-            throw new Error("Unable to get address for new categorical market.");
-        }
-        await marketCreation.createMarket.bind({ value: marketCreationFee })(universe.address, endTime, 2, feePerEthInWei, denominationToken.address, numTicks, designatedReporterAddress);
-        const market = await parseAbiIntoMethods(this.ethjsQuery, this.signatures["Market"], { to: marketAddress, from: this.testAccounts[0], gas: "0x5b8d80" });
-        const marketNameHex = padAndHexlify("Market", 64, "right");
-        if (await market.getTypeName() !== marketNameHex) {
-            throw new Error("Unable to create new categorical market");
-        }
-        return market;
-    }
-
     private async createReasonableBinaryMarket(universe, denominationToken): Promise<Contract> {
         const block = await this.ethjsQuery.getBlockByNumber(0, true);
         const blockDateTime = await this.parseBlockTimestamp(block.timestamp);
         const blockDateTimePlusDay = new Date();
         blockDateTimePlusDay.setDate(blockDateTime.getDate() + 1);
-        return await this.createBinaryMarket(universe, blockDateTimePlusDay.getTime()/1000, 10 ** 16, denominationToken, this.testAccounts[0], 10 ** 18);
+        return await this.createMarket(universe, 2, blockDateTimePlusDay.getTime()/1000, 10 ** 16, denominationToken, this.testAccounts[0], 10 ** 4);
     }
 
     private async createReasonableCategoricalMarket(universe, numOutcomes, denominationToken): Promise<Contract> {
         const block = await this.ethjsQuery.getBlockByNumber(0, true);
         const blockDateTime = await this.parseBlockTimestamp(block.timestamp);
         const blockDateTimePlusDay = new Date();
         blockDateTimePlusDay.setDate(blockDateTime.getDate() + 1);
-        return await this.createCategoricalMarket(universe, numOutcomes, blockDateTimePlusDay.getTime()/1000, 10 ** 16, denominationToken, this.testAccounts[0], 3 * 10 ** 17);
+        return await this.createMarket(universe, numOutcomes, blockDateTimePlusDay.getTime()/1000, 10 ** 16, denominationToken, this.testAccounts[0], numOutcomes * 10**4);
     }
 
     private async createReasonableScalarMarket(universe, priceRange, denominationToken): Promise<Contract> {
         const block = await this.ethjsQuery.getBlockByNumber(0, true);
         const blockDateTime = await this.parseBlockTimestamp(block.timestamp);
         const blockDateTimePlusDay = new Date();
         blockDateTimePlusDay.setDate(blockDateTime.getDate() + 1);
-        return await this.createScalarMarket(universe, blockDateTimePlusDay.getTime()/1000, 10 ** 16, denominationToken, 40 * 10 ** 18, this.testAccounts[0]);
+        return await this.createMarket(universe, 2, blockDateTimePlusDay.getTime()/1000, 10 ** 16, denominationToken, this.testAccounts[0], 10**4);
```

Comment:
It seems like this function is doing the exact same thing as `createReasonableBinaryMarket`.  Is that supposed to be the case now after the `numTicks` changes?  If so, maybe one of these functions should just call the other?

Reply:
Ah, good point.  Binary markets and scalar markets are no longer different aside from potentially the scale of `numTicks`.  I'll delete one of them.  I may actually reduce down to a single `createReasonableMarket` that just takes in `numOutcomes`....